.github/workflows/update-sdk.yml

```diff
@@ -13,23 +13,18 @@ jobs:
       - uses: actions/setup-go@v4
         with:
           go-version-file: 'go.mod'
-      - name: update
-        run: 	| 
-          go install github.com/icholy/gomajor@latest 
-          make update-atlas-sdk
+      - name: Update files
+        run:  make tools update-atlas-sdk
```

Comment:
[Nit] Change adds to the build time. Ok to leave it.

Reply:
i know, but just a few seconds as a tradeoff for consistency and remove duplication